Gemfile

```diff
@@ -66,6 +66,7 @@ gem 'terser', '~> 1.1'
 
 # we use the git version of acts_as_versioned, and need to include it in this Gemfile
 gem 'acts_as_versioned', git: 'https://github.com/technoweenie/acts_as_versioned.git'
+gem 'foodsoft_b85', path: 'plugins/b85'
```

Comment:
This means the plugin is enabled by default, but [here](https://github.com/foodcoops/foodsoft/pull/1173#issuecomment-2995968924) we agreed it shouldn't be.
So I suggest this PR to be marked **draft** for now, lest anyone should review, right?

Reply:
I marked it as _draft_ as I'm still working on it.

Regarding enabled state: What actually means *enabled*? In Foodsoft, there are two ways to *disable* a plugin:
- soft disable: most plugins support a config flag (typically called `use_<plugin-name>`) which can be set to `false`
- hard disable: removing the plugin from `Gemfile`

I'm not sure if there are any hard-disabled plugins in Foodsoft yet. It would mean, we cannot run the tests easily.

Reply:
Yes, very good, thank you for your answer. In fact, I had overlooked the deactivated plugins in Gemfile, sorry. So I moved the plugin to the test section, that seems to work fine.